src/DirectusProvider.tsx

```diff
@@ -144,14 +144,14 @@ export const DirectusProvider = <T extends TypeMap = TypeMap>({
           })) as UserType;
 
           if (dUser) {
-            newAuthState = 'authenticated';
+            newAuthState = AuthStates.AUTHENTICATED;
             setUser(dUser);
           }
         }
       } catch (error) {
         console.log('auth-error', error);
```

Comment:
Can we print an useful message here?

Reply:
That's not realy neccessery, we could even drop the console log. The only time an error ocures there is if autoLogin is set to true but no valide auth token is stored.

Reply:
Are you suggesting to remove the try/catch and just call directus API right?

Reply:
No, I would sugest to remove the console.log from the catch.

Reply:
So any error would be ignored?

Reply:
Yes

Reply:
Let's solve this before merge. If I understand correcly, with `autoLogin` enabled (which isn't documented at all) that code is executed, calling Directus refresh is called. If something go wrong we don't provide any message and/or log nothing?

I'm trying to also understand this https://docs.directus.io/reference/old-sdk.html#basic-authentication ... where the token come from? This is also undocumented I think...

Reply:
Thank you for your work and the explanation, now I understand. The question how to handle the `refresh` call correcly remains (could be a simple network error).

Reply:
We could include a onAutoLoginError funktion?

Reply:
I'm not a React guru 😄  and I can't really answer to that question, in addition it's an async errors so it would be impossibile to catch with something like https://github.com/bvaughn/react-error-boundary.

Adding `onAutoLoginError` prop seems appropriate as user can handle it as he wish.